src/libs/actions/IOU.ts

```diff
@@ -14645,17 +14645,91 @@ function updateSplitTransactions({
                 splitApiParams[`splits[${i}][${key}]`] = value !== null && typeof value === 'object' ? JSON.stringify(value) : value;
             }
         }
-        const parameters: SplitTransactionParams = {
+
+        if (isCreationOfSplits) {
+            const isTransactionOnHold = isOnHold(originalTransaction);
+            const holdReportActionIDs: string[] = [];
+            const holdReportActionCommentIDs: string[] = [];
```

Comment:
Let's declare them inside `isTransactionOnHold` as they're only being used there if that's the case.